recipe/meta.yaml

```diff
@@ -14,7 +14,7 @@ source:
 
 build:
   merge_build_host: True  # [win]
-  number: 0
+  number: 1
```

Comment:
You probably don't need to bump the version number for changes like these. The recipe maintainers metadata is for conda-forge to grant write access to this repo. In other words, it doesn't affect the package binary that will be created, so it's not required to upload a new binary to Anaconda Cloud.

That being said, it doesn't hurt anything besides having an extra file stored in the cloud.

Reply:
thanks! The PR template has

`[ ] Bumped the build number (if the version is unchanged)`

so I assumed that if I didn't bump it some magic would happen and the CI stuff would complain and say something like "hey `meta.yml` is in the diff but you didn't change the build number".